epic.py

```
@@ -128,46 +125,54 @@ def rotate_photos(num_photos, rotate_delay, blend_enabled=False, blend_time=5):
+    last_check = datetime.datetime.now() - datetime.timedelta(hours=1)
+    num_photos = 0
+
+    while running:
+        # Did anyone try to quit the app?
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                running = False
+                pygame.quit()
+
+        # If we haven't checked for new images recently, check for new images
+        if last_check < datetime.datetime.now() - datetime.timedelta(minutes=check_delay) or first_run == True:
```

Ruff (0.14.14)

[error] Avoid equality comparisons to `True`; use `first_run:` for truth checks

Replace with `first_run`

(E712)